### PR TITLE
Gm templates Winter

### PR DESCRIPTION
**Pull request by targetingsnake**
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Added winter template for GM, maybe desert will follow here soon

### Please specify which Issue this PR Resolves.
none

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

1. Build mod
2. load with mods for GM
3. test winter template

********************************************************
Notes:

### CODE REVIEW

**Comment by targetingsnake** [comment]
Desert Camo will be added soon, put this already up so winter stuff can be reviewed

**Comment by targetingsnake** [comment]
desert camo will be delayed till 3.3 or so

**Comment by Bob-Murphy** [comment]
Also, once done please adapt the title and the description accordingly.
Thank you

**Comment by targetingsnake** [comment]
@Bob-Murphy done

**Review by killerswin2 — APPROVED** [review]
This is ready for merge and testing.

File: A3A/addons/core/Templates/Templates.hpp [hunk 1]
```diff
@@ -671,6 +681,16 @@ class Templates
...
         name = "Bundeswehr";
         file = "GM_AI_BW";
+        climate[] = {"temperate"};
+    };
+
+    class GM_BW_Arctic : GM_Base
+    {
+        side = "Occ";
+        flagTexture = "\gm\gm_core\data\flags\gm_flag_ge_co";
+        name = "Bundeswehr Winter";
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates.hpp:691]: Please add the GM Prefix to the Name, so people know what is what.

Add Temperate in the name of the "Old" ones, so people know what they select.

add Tropical and Arid to the Climate for the "old" ones, so they arent hidden by default in the Camo selection.

File: A3A/addons/core/Templates/Templates/GM/GM_AI_BW_arctic.sqf [hunk 2]
```diff
@@ -0,0 +1,871 @@
...
+["vehiclesTanks", ["gm_dk_army_m113a2dk_win", "gm_dk_army_m113a2dk_un", "gm_ge_army_leopard1a1a1_wiw", "gm_ge_army_leopard1a1a2_wiw", "gm_ge_army_leopard1a1a3_wiw", "gm_ge_army_leopard1a1a4_wiw", "gm_ge_army_leopard1a3_wiw", "gm_ge_army_leopard1a3a1_wiw", "gm_ge_army_leopard1a3a2_wiw", "gm_ge_army_leopard1a3a3_wiw", "gm_ge_army_leopard1a5_wiw", "gm_ge_army_leopard1a5a1_wiw", "gm_ge_army_leopard1a1a1_win", "gm_ge_army_leopard1a1a2_win", "gm_ge_army_leopard1a1a3_win", "gm_ge_army_leopard1a1a4_win", "gm_ge_army_leopard1a3_win", "gm_ge_army_leopard1a3a1_win", "gm_ge_army_leopard1a3a2_win", "gm_ge_army_leopard1a3a3_win", "gm_ge_army_leopard1a5_win", "gm_ge_army_leopard1a5a1_win", "gm_ge_army_leopard1a1a1_un", "gm_ge_army_leopard1a1a2_un", "gm_ge_army_leopard1a1a3_un", "gm_ge_army_leopard1a1a4_un", "gm_ge_army_leopard1a3_un", "gm_ge_army_leopard1a3a1_un", "gm_ge_army_leopard1a3a2_un", "gm_ge_army_leopard1a3a3_un", "gm_ge_army_leopard1a5_un", "gm_ge_army_leopard1a5a1_un"]] call _fnc_saveToTemplate;
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/GM/GM_AI_BW_arctic.sqf:29]: Kill the UN Variants if there are Winter variants.

File: A3A/addons/core/Templates/Templates/GM/GM_AI_NVA_arctic.sqf [hunk 3]
```diff
@@ -0,0 +1,878 @@
...
+["equipmentBox", "Box_NATO_Equip_F"] call _fnc_saveToTemplate; //Changeing this from default will require you to define logistics attachement offset for the box type
+
+["vehiclesBasic", ["gm_gc_army_p601"]] call _fnc_saveToTemplate;
+["vehiclesLightUnarmed", ["gm_gc_army_uaz469_cargo_un", "gm_gc_army_uaz469_cargo_win", "gm_gc_army_uaz469_cargo_un", "gm_gc_army_uaz469_cargo_win", "gm_gc_army_btr60pu12_un", "gm_gc_army_btr60pu12_win", "gm_gc_army_btr60pu12_olw", "gm_gc_army_uaz469_cargo_olw"]] call _fnc_saveToTemplate;
+["vehiclesLightArmed",["gm_gc_army_uaz469_dshkm_win", "gm_gc_army_uaz469_dshkm_un", "gm_gc_army_brdm2_win", "gm_gc_army_brdm2_un", "gm_gc_army_brdm2_olw", "CUP_O_GAZ_Vodnik_BPPU_RU", "CUP_O_GAZ_Vodnik_KPVT_RU"]] call _fnc_saveToTemplate;
+["vehiclesTrucks", ["gm_gc_army_btr60pa_win", "gm_gc_army_btr60pa_un", "gm_gc_army_ural375d_cargo_un", "gm_gc_army_ural4320_cargo_un", "gm_gc_army_ural4320_cargo_win", "gm_gc_army_ural375d_cargo_win", "gm_gc_army_ural375d_cargo_olw", "gm_gc_army_ural4320_cargo_olw", "gm_gc_army_btr60pa_olw"]] call _fnc_saveToTemplate;
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/GM/GM_AI_NVA_arctic.sqf:23]: Same here,
Kill the UN Variants if there are Winter variants.

File: A3A/addons/core/Templates/Templates/GM/GM_Vehicle_Attributes.sqf [hunk 4]
```diff
@@ -4,23 +4,56 @@
     ["CUP_O_BMP3_RU", ["cost", 120], ["threat", 150]],
     ["CUP_O_BMP2_RU", ["cost", 120], ["threat", 150]],
     ["gm_gc_army_bmp1sp2", ["cost", 120], ["threat", 150]],
+    ["gm_gc_army_bmp1sp2_olw", ["cost", 120], ["threat", 150]],
```
Comment (Lazejun) [inline A3A/addons/core/Templates/Templates/GM/GM_Vehicle_Attributes.sqf:7]: Same UN Cleanup

**Review by Lazejun — CHANGES_REQUESTED** [review]

File: A3A/addons/core/Templates/Templates/GM/GM_AI_BW_arctic.sqf (hunk 2 above)
Comment (Bob-Murphy) [reply A3A/addons/core/Templates/Templates/GM/GM_AI_BW_arctic.sqf:29]: None of the UN variants have any UN markings. They are plain white.
image

File: A3A/addons/core/Templates/Templates/GM/GM_AI_NVA_arctic.sqf (hunk 3 above)
Comment (Bob-Murphy) [reply A3A/addons/core/Templates/Templates/GM/GM_AI_NVA_arctic.sqf:23]: same as above, they are white without UN markings

File: A3A/addons/core/Templates/Templates/GM/GM_Vehicle_Attributes.sqf (hunk 4 above)
Comment (Bob-Murphy) [reply A3A/addons/core/Templates/Templates/GM/GM_Vehicle_Attributes.sqf:7]: same as above, they are white without UN markings

**Comment by Bob-Murphy** [comment]
btw, changed `Winter` to `Arctic` to have it matching the other stuff we have

**Review by Lazejun — APPROVED** [review]
Good for now

**Pull request merged**